src/bp-forums/replies/functions.php

```diff
@@ -1910,13 +1910,19 @@ function bbp_trashed_reply( $reply_id = 0 ) {
  * @uses bbp_is_reply() To check if the passed id is a reply
  * @uses do_action() Calls 'bbp_untrashed_reply' with the reply id
  */
-function bbp_untrashed_reply( $reply_id = 0 ) {
+function bbp_untrashed_reply( $reply_id = 0, $previous_status ) {
 	$reply_id = bbp_get_reply_id( $reply_id );
 
 	if ( empty( $reply_id ) || ! bbp_is_reply( $reply_id ) ) {
 		return false;
 	}
 
+	$update_reply = array(
+		'ID' 			=> $reply_id,
+		'post_status'	=>	$previous_status
+	);
```

Comment:
@ngolwalkar make the code proper alligned

Reply:
@chetansatasiya It is done now. Please check.